infrastructure/terraform/alpha/lag01/tasks/ingress/scripts/helmInstallNginxIngress.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+IFS=$'\n\t'
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I know this is not a review and this code will be thrown away, but the variable `$SCRIPT_DIR` is not being used in this script